src/components/Badges/JourneySideSheet.vue

```diff
@@ -123,3 +124,15 @@ const journeyCtaBtn = computed(() => {
 });
 
 </script>
+
+<style lang="postcss" scoped>
+
+.sidesheet-journey :deep(h2) {
```

Comment:
Maybe we should be making these changes to the side sheet component? Is the journey the only place where these two changed components are visible for the side sheet currently?

Reply:
The impact dashboard should be just the BP, so if the BP looks good there in the side sheet, changes should be safe. In that case, keep the ticket in progress until the changes are reviewed on dev with product, so you can finish up the shared component changes. Thanks!